README.md

```diff
@@ -1,3 +1,22 @@
+# Spectrometer
+
+Spectrometer is a polyglot dependency analysis tool, designed to identify and
+report dependency trees for many languages and package managers.  See the
+[docs](docs/strategies.md) for a non-exhaustive list of supported
+languages/managers.
+
+```sh
+me@mydesk:~/myproject$ fossa2 analyze -o
```

Comment:
We should rename this to `fossa` before we merge this `README` cc @cnr.

Reply:
I genuinely don't know what to do about the naming, because I suspect in some cases customers may need to use both the old version and the new version -- at least until we have support for archive uploads and the languages from v1

Reply:
Will they ever use both in the same environment? I don't think so, and we can instruct SE to specifically avoid integrating that way. Is there a likely I-need-both scenario that I'm missing?

Reply:
I generally agree with Leo here, but I will have a quick chat with an SE to make sure that it makes sense to them as well.